CarRentalCompany.Infrastructure/Repositories/ActivityInstanceRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using CarRentalCompany.Domain.Models;
+using CarRentalCompany.Domain.Repositories;
+using ActivityInstanceDb = CarRentalCompany.Infrastructure.Entities.ActivityInstance;
+
+namespace CarRentalCompany.Infrastructure.Repositories
+{
+    internal class ActivityInstanceRepository : IActivityInstanceRepository
+    {
+        private readonly CarRentalCompanyDbContext _context;
+
+        public ActivityInstanceRepository(CarRentalCompanyDbContext context)
+        {
+            _context = context;
+        }
+
+        public void Add(IEnumerable<ActivityInstance> activities)
+        {
+            var activitiesDb = activities
+                .Select(activity => ActivityInstanceDb.Create
```

Comment:
Skoro już robisz na ActivityInstanceDb metody Create, która twoorzy to na podstawie model domenowego + masz metode, która na podstawie DB tworzy domenowy to w sumie pytanie czemu nie zrobisz w ActivityInstanceDB metody create, która przyjmuje model domenowy. Przekazujesz to jako osobne parametry, gdzie możesz zrobić potencjalnie babola, a przecież model DB i tak wie o domenowy to po co dodatkowe miejsce problemu?